virttest/test_setup.py

```diff
@@ -1313,6 +1303,23 @@ def get_pf_mlx(self):
             cmd, shell=True, timeout=60).splitlines()
         return (PF_devices)
 
+    def get_pf_ids(self):
+        """
+        Get the PF devices on x86 host
+        """
+        cmd = "lspci | grep -v 'Virtual Function' |awk '/%s/ {print $1}'" % self.pf_filter_re
```

Comment:
Because some vfs may contain the key words which is used to filter the pf.  
e.g. 
PF: Broadcom Limited NetXtreme II BCM57810 10 Gigabit Ethernet (rev 10)
VF: Broadcom Limited NetXtreme II BCM57810 10 Gigabit Ethernet Virtual Function